_posts/release/2018-05-08-accumulo-1.9.1.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Apache Accumulo 1.9.1
+draft: true
+---
+
+Accumulo 1.9.1 contains multiple bug fixes for write ahead log recovery. Write
+ahead log recovery is the process of restoring data that was in memory when a
+tablet server died. These bugs could lead to data loss and are present in
+1.8.0, 1.8.1, and 1.9.0.  Because the bugs can negatively impact Accumulo's
+metadata table, even users that mainly use bulk import may be affected.  It is
```

Comment:
Could make this sentence more clear... "Systems that mainly use bulk import for ingest are also affected since the bugs can negatively impact the metadata table."